src/Pages/Home.js

```diff
@@ -74,6 +76,76 @@ function Home(){
                     <button className="Font_style_2">Resume</button>
                 </div>
             </div> 
+            {/* Project section */}
```

Comment:
This is becoming a very large file and may be confusing to anyone viewing your code. How about breaking up the Projects section into it’s own component and just imp[lementing that component here on this page?